docs/reference/software-versions.mdx

```diff
@@ -14,6 +12,50 @@ Release candidates are software releases that are also released to the [Testnet]
 
 [testnet]: ./networks.mdx
 
+## Protocol 21 (Mainnet, June 18, 2024)
+
+### Software
+
+| Software | Version |
+| --- | --- |
+| XDR |  |
+| Rust XDR |  |
+| Soroban Environment |  |
+| Stellar Core |  |
+| Soroban Rust SDK |  |
+| Soroban CLI |  |
+| Soroban RPC |  |
+| Stellar Horizon |  |
+| Stellar Quickstart |  |
+| Stellar JS Stellar Base |  |
+| Stellar JS Stellar SDK |  |
+| Freighter |  |
+| Laboratory |  |
+| Soroban React Payment dapp | `TBD` |
+| Soroban Mint Token dapp | `TBD` |
+| Soroban Swap Token dapp | `TBD` |
+| Futurenet Network Passphrase | `Test SDF Future Network ; October 2022` |
+| Testnet Network Passphrase | `Test SDF Network ; September 2015` |
+| Mainnet Network Passphrase | `Public Global Stellar Network ; September 2015` |
+
+### Release notes
+
+#### Core
+
+New features in Protocol 21:
+
+- Secp256r1 support in Soroban host: https://github.com/stellar/stellar-protocol/blob/master/core/cap-0051.md
+- Soroban host function for extending TTL of contract instance and code separately: https://github.com/stellar/stellar-protocol/blob/master/core/cap-0053.md
+- Use refined cost model for VM instantiation in order to reduce the VM instantiation metered costs: https://github.com/stellar/stellar-protocol/blob/master/core/cap-0054.md
+- Intra-transaction VM module caching for the further Soroban cost reduction: https://github.com/stellar/stellar-protocol/blob/master/core/cap-0055.md, https://github.com/stellar/stellar-protocol/blob/master/core/cap-0056.md
+
+#### Soroban Rust SDK
+
+Key protocol-related changes:
+
+- Support for secp256r1 signature verification
+- Support for extending TTL of contract instance and code separate from each other
+
 ## Protocol 21 (Testnet only, May 20, 2024)
 
 ### Software
```

Comment:
How about copying these versions over and we can have each team update to the latest? From Core side, XDR, Rust XDR, Soroban Env, Stellar Core are the same. Only Soroban Rust SDK needs to be updated to [21.1.0-rc.1](https://github.com/stellar/rs-soroban-sdk/releases/tag/v21.1.0-rc.1)